lms/djangoapps/lti_provider/models.py

```diff
@@ -0,0 +1,14 @@
+"""
+Database models for the LTI provider feature.
+"""
+from django.db import models
+
+
+class LtiConsumer(models.Model):
```

Comment:
It's a nitpick, but acronyms are generally kept capitalized in class names (so `LTIConsumer`).